_includes/sections/vpn.html

```diff
@@ -24,10 +24,12 @@ <h5><span class="badge badge-success">406+ Servers</span></h5>
     <h5><span class="badge badge-success">WireGuard Support</span></h5>
     <p>In addition to standard OpenVPN connections, Mullvad supports WireGuard. WireGuard is an experimental protocol with theoretically better security and higher reliability, although it is not currently recommended for production use.</p>
     <h5><span class="badge badge-success">Independently Audited</span></h5>
-    <p>Mullvad's VPN clients have been audited by Cure53 and Assured AB in a pentest report <a href="https://cure53.de/pentest-report_mullvad_v2.pdf">published at cure53.de</a>. The security researchers concluded:</p>
+    <p>Mullvad's VPN clients have been audited by Cure53 and Assured AB in a pentest report <a href="https://cure53.de/pentest-report_mullvad_v2.pdf">published at cure53.de</a> and . The security researchers concluded:</p>
```

Comment:
```suggestion
    <p>Mullvad's VPN clients have been audited by Cure53 and Assured AB in a pentest report <a href="https://cure53.de/pentest-report_mullvad_v2.pdf">published at cure53.de</a>. The security researchers concluded:</p>
```

What is this ` and .` ?

Reply:
My bad. I was looking for the Assured AB report but it seems the report was a jointly done one which was published by Cure53.